tools/download-deps.sh

```diff
@@ -98,6 +103,19 @@ download_dependency() {
     "${fetch}" "${tool}" "${url_and_path}" "${custom_bindir:-$bin_dir}"
 }
 
+validate_flux_binary() {
+    cd ../pkg/flux/bin
+    local flux_hash
+    hash_result=$(openssl dgst -sha256 flux)
```

Comment:
You might be able to simplify this with `if ! grep $((openssl dgst -sha256 flux | cut -d ' ' -f 2) fluxchecksums.txt`...